docs/UserGuide.md

```diff
@@ -3,34 +3,74 @@ layout: page
+This document is a *User Guide* for Athena, a contact- and event- management desktop application. For more details on
+the suitability and functionalities of the application for your usage, please refer to the [Introduction](#Introduction)
+section below.
+
+#### Table of Contents
+
+* [Introduction](#Introduction)
+* [Quick Start](#Quick-start)
+* [Features](#Features)
+    * [General](#General)
+        * [`help`](#viewing-help) - viewing help
+        * [`exit`](#exiting-the-program--exit) - exiting the program
+    * [Contact](#Contact)
+        * [`add`](#adding-a-contact-add) - adding a contact
+        * [`clear`](#clearing-all-contacts--clear) - clearing all contacts
+        * [`delete`](#deleting-a-contact--delete) - deleting a contact
+        * [`edit`](#editing-a-contact--edit) - editing a contact
+        * [`find`](#finding-a-contact--find) - finding a contact
+        * [`list`](#listing-all-contacts--list) - listing a contact
+        * [`sort`](#sorting-displayed-contacts--sort) - sorting displayed contacts
+    * [Event](#event)
+        * [`add`](#adding-an-event-add) - adding an event
+        * [`clear`](#clearing-all-events--clear) - clearing all events
+        * [`delete`](#deleting-an-event--delete) - deleting an event
+        * [`edit`](#editing-an-event--edit) - editing an event
+        * [`find`](#finding-an-event--find) - finding an event
+        * [`list`](#listing-all-events--list) - listing an event
+* [FAQ](#FAQ)
+* [Command Summary](#Command-summary)
 
-* Table of Contents
-{:toc}
+---
 
---------------------------------------------------------------------------------------------------------------------
+## Introduction
+
+Athena is a **desktop app for managing contacts and events, optimized for use via a Command Line Interface** (CLI) while
+still having the benefits of a Graphical User Interface (GUI).
+
+Athena is perfect for you if you:
+* need to categorise a huge number of contacts or events,
+* need to keep track of the people who attend specific events,
+* can type fast!
 
+
+---
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your computer.
 
-1. Download the latest `Athena.jar` (no releases yet).
+1. Download the latest `Athena.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-4/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for Athena.
 
-1. Double-click the file or run it with `java -jar Athena.jar` to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file or run it with `java -jar Athena.jar` to start the app. The GUI similar to the below should
+   appear in a few seconds. Note how the app contains some sample data.<br>
```

Comment:
"A window similar to the one below should..." instead of "The GUI similar to the below should..."